02-microarray/clustering/clustering_example.Rmd

```diff
@@ -1,20 +1,97 @@
 ---
-title: "Clustering Data"
-author: "ALSF CCDL - Candace Savonen"
+title: "Clustering Heatmap - Microarray"
+author: "CCDL for ALSF"
 output:   
   html_notebook: 
     toc: true
     toc_float: true
 ---
 
-*Purpose*: This notebook illustrates one way that you can use harmonized data 
-from refine.bio in downstream analyses.
+# Purpose of the analysis:
 
-## 1) Install libraries
-This script uses the bioconductor R package ComplexHeatmap for clustering and 
-creating a heatmap.
-<br/> <i> Citation:</i> Gu Z, Eils R, Schlesner M (2016). “Complex heatmaps 
-reveal patterns and correlations in multidimensional genomic data.” Bioinformatics.
+Clustering can help you investigate which samples (or genes) are more related to each other.
+Here we use the Bioconductor R package `ComplexHeatmap` for clustering and creating a heatmap.
+In this example, the gene expression matrix is subset to only include genes with high variance.
+This set of genes is then used to create a heatmap.
+
+## Further learning resources about this analysis
+
+ - [Clustering strategies](https://www.statsandr.com/blog/clustering-analysis-k-means-and-hierarchical-clustering-by-hand-and-in-r/)  
+ - [Complex Heatmap book](https://jokergoo.github.io/ComplexHeatmap-reference/book/)  
+ - [Complex Heatmap vignette](https://bioconductor.riken.jp/packages/3.4/bioc/vignettes/ComplexHeatmap/inst/doc/s1.introduction.html)  
+ - [Complex Heatmap publication](https://academic.oup.com/bioinformatics/article/32/18/2847/1743594)  
+ 
+***
+
+# How to run this example
+
```

Comment:
We will need to tell them what they need to have setup and we should link them to `What you need to install to run the examples` section in `Getting Started`

Reply:
Good call. Forgot about that!

Reply:
Related https://github.com/AlexsLemonade/refinebio-examples/issues/109#issuecomment-656827906